DrivewayDentDeletion/prereqs.sh

```diff
@@ -95,21 +127,23 @@ oc create secret generic -n ace ace-ddd-dev-creds \
   --from-file=policyDescriptor=/tmp/policyDescriptor.xml \
   --dry-run -o yaml | oc apply -f -
 
-oc exec -n postgres -it $(oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}') \
-  -- psql -U admin -d sampledb -c \
-'CREATE TABLE QUOTES (
-  QuoteID SERIAL PRIMARY KEY NOT NULL,
-  Name VARCHAR(100),
-  EMail VARCHAR(100),
-  Address VARCHAR(100),
-  USState VARCHAR(100),
-  LicensePlate VARCHAR(100),
-  ACMECost INTEGER,
-  ACMEDate DATE,
-  BernieCost INTEGER,
-  BernieDate DATE,
-  ChrisCost INTEGER,
-  ChrisDate DATE);'
-
-oc create secret generic cluster-kubeconfig --from-file=kubeconfig=/root/kubeconfig.yaml
-oc create secret generic task-helm-tls --from-file=key=$HELM_HOME/key.pem --from-file=cert=$HELM_HOME/cert.pem --from-file=ca=ca.pem --dry-run -o yaml | oc apply -f -
+echo "Checking if postgres pod name exists before configuring database"
+oc get pod -n postgres -l name=postgresql -o jsonpath='{.items[].metadata.name}' 2> /dev/null
```

Comment:
Nice idea, but probably better to wait for the pod to be ready and exit with error if the pod not ready within a timeout.

Reply:
How long does it take for the pods to get to ready? Shouldn't we have that wait in the install step where the postgres is being installed instead of pre-reqs where the assumption is everything executed before this file is good and running?

Reply:
Asset repo setup waits for asset repo (released by the release-products script) to be ready. I think it's fair for this script to wait for postgres (also released by the release-products script), to be ready. Postgres is pretty fast starting, so likely it would be ready, but it's a good idea to make sure.